webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div class="VuetifyApp">
     <div v-show="loading">
       <div class="VuetifyApp loadingComposer">
```

Comment:
do you need really the class VuetifyApp in each block ?
If you keep it only on the parent div, will it be sufficient ?